src/main/java/com/bmc/truesight/remedy/App.java

```diff
@@ -133,32 +141,67 @@ private static void readAndIngest(ARServerForm form) {
+        } catch (RemedyLoginFailedException e) {
+            log.error("Login Failed : {}", e.getMessage());
+        } catch (RemedyReadFailedException e) {
+            if (lastEventList.isEmpty()) {
+                log.error("Reading tickets from Remedy server failed for StartDateTime, EndDateTime ({},{}). Please try running the script after some time with the same timestamps.", new Object[]{e.getMessage(), ScriptUtil.dateToString(config.getStartDateTime()), ScriptUtil.dateToString(config.getEndDateTime())});
+            } else {
+                log.error("Due to some issue Reading tickets from Remedy server failed for StartDateTime, EndDateTime ({},{}). Please try running the script after some time from the last successful timestamp as below.", new Object[]{ScriptUtil.dateToString(config.getStartDateTime()), ScriptUtil.dateToString(config.getEndDateTime())});
+                Date createdDate = convertToDate(lastEventList.get((lastEventList.size() - 1)).getCreatedAt());
+                Date modDate = convertToDate(lastEventList.get((lastEventList.size() - 1)).getProperties().get("LastModDate"));
+                Date closedDate = convertToDate(lastEventList.get((lastEventList.size() - 1)).getProperties().get("ClosedDate"));
+                log.error("Created Date : {}", new Object[]{createdDate});
+                if (lastEventList.get((lastEventList.size() - 1)).getProperties().get("LastModDate") != null) {
+                    log.error("Last Modified Date : {}", new Object[]{modDate});
+                }
+                if (lastEventList.get((lastEventList.size() - 1)).getProperties().get("ClosedDate") != null) {
+                    log.error("Closed Date : {}", new Object[]{closedDate});
+                }
+            }
+        } catch (BulkEventsIngestionFailedException e) {
+            if (lastEventList.isEmpty()) {
+                log.error("Ingestion Failed (Reason : {}) for StartDateTime, EndDateTime ({},{}). Please try running the script after some time with the same timestamps.", new Object[]{e.getMessage(), ScriptUtil.dateToString(config.getStartDateTime()), ScriptUtil.dateToString(config.getEndDateTime())});
+            } else {
+                log.error("Ingestion Failed (Reason : {}) for StartDateTime, EndDateTime ({},{}). Please try running the script after some time from the last successful timestamp as below.", new Object[]{e.getMessage(), ScriptUtil.dateToString(config.getStartDateTime()), ScriptUtil.dateToString(config.getEndDateTime())});
+                Date createdDate = convertToDate(lastEventList.get((lastEventList.size() - 1)).getCreatedAt());
+                Date modDate = convertToDate(lastEventList.get((lastEventList.size() - 1)).getProperties().get("LastModDate"));
+                Date closedDate = convertToDate(lastEventList.get((lastEventList.size() - 1)).getProperties().get("ClosedDate"));
+                log.error("Created Date : {}", new Object[]{ScriptUtil.dateToString(createdDate)});
```

Comment:
These don't necessarily have to be .error messages, the above (line 192 is .error and correct)... can change them to .info